AISKU/snippet/snippet.js

```diff
@@ -31,14 +31,39 @@
                 // Invoke the real method with the captured original arguments
                 appInsights[name].apply(appInsights, originalArguments);
             });
-        }
-    };
+        };
+    }
 
-    var method = ["PageView"];
+    var method = ["PageView", "Exception"];
     while (method.length) {
         createLazyMethod("track" + method.pop());
     }
 
+    // Collect global errors
+    if (aiConfig.extensionsConfig &&
+        aiConfig.extensionsConfig.ApplicationInsightsAnalytics &&
```

Comment:
use aiConfig.extensionsConfig[this.identifier] as configurations are by plugin identifier

Reply:
or add comment here

Reply:
I've added a comment to clarify 7c915f3